packages/esm-bed-management-app/src/bed-administration/bed-administration-form.component.tsx

```diff
@@ -232,8 +241,8 @@ const BedAdministrationForm: React.FC<BedAdministrationFormProps> = ({
                     {occupancyStatuses.map((occupancyStatus, index) => (
                       <SelectItem
                         key={`occupancyStatus-${index}`}
-                        text={t('occupancyStatus', `${occupancyStatus}`)}
-                        value={t('occupancyStatus', `${occupancyStatus}`)}
+                        text={t(`occupancyStatus${occupancyStatus}`, `${occupancyStatus}`)}
+                        value={t(`occupancyStatus${occupancyStatus}`, `${occupancyStatus}`)}
```

Comment:
The value of the item should not be translated, should it?  Isn't this what is sent to the server?  Only the displayed value should be translated.